src/util/tev_curl.c

```diff
@@ -183,9 +183,9 @@ static void handle_curlmsg_done(CURLMsg *message)
     CURL *easy_handle;
     CURLcode crv;
     struct tevent_req *req;
+    int response_code = 0;
```

Comment:
gcc + libcurl-devel-7.53.1-3 does not like this type:
```
In function ‘handle_curlmsg_done’,
    inlined from ‘process_curl_activity.isra.0’ at ../sssd/src/util/tev_curl.c:245:13:
../sssd/src/util/tev_curl.c:224:11: error: call to ‘_curl_easy_getinfo_err_long’ declared with attribute warning: curl_easy_getinfo expects a pointer to long for this info [-Werror]
     crv = curl_easy_getinfo(easy_handle, CURLINFO_RESPONSE_CODE, &response_code);
           ^~~~~~~~~~~~~~~~~
```

changing type to long fixed the warning.